gems/sorbet-runtime/test/types/props/serializable.rb

```diff
@@ -152,6 +152,8 @@ class ChildWithDefault < ParentWithNoDefault
       assert_equal(MySerializable, storytime[:klass])
       assert_equal(:foo, storytime[:prop])
       assert_equal("Won't respond like hash", storytime[:value])
+
+      T::Configuration.soft_assert_handler = nil
```

Comment:
Can we do this in an `ensure`? if one of the asserts fails, we won't have reset the soft_assert_handler, and it will be wrong for the other tests.